evaluation/src/eval_dart.py

```diff
@@ -12,18 +12,11 @@ def eval_script(path: Path):
             "stderr": r.stderr,
         }
 
-    # todo: scrape for the various errors in stderr, stdout
     r = run(["dart", str(path)], timeout_seconds=15)
     if r.timeout:
         status = "Timeout"
     elif r.exit_code == 0:
         status = "OK"
-    elif "ERR_ASSERTION" in r.stderr:
-        status = "AssertionError"
-    elif "SyntaxError" in r.stderr:
-        status = "SyntaxError"
-    elif "ReferenceError" in r.stderr:
-        status = "ReferenceError"
```

Comment:
At this point, for benchmarking, it only really matters that you classify each run as pass or fall. The MultiPL-E paper had a finer-grained analysis of the types of errors, which is what this stuff was for.  So, this fine-grained categorization is optional at this point.